Libraries/test/Amazon.Lambda.RuntimeSupport.Tests/Amazon.Lambda.RuntimeSupport.IntegrationTests/BaseCustomRuntimeTest.cs

```diff
@@ -143,6 +143,7 @@ private async Task CreateBucketWithDeploymentZipAsync(IAmazonS3 s3Client, string
                     BucketName = bucketName
                 };
                 await s3Client.PutBucketAsync(putBucketRequest);
+                await Task.Delay(10000);
```

Comment:
Love the arbitrary delays ;)

Reply:
A better fix would be periodically checking the existence of the bucket with a timeout then using a constant delay which can never be trusted 100%.